editoast/src/models/stdcm_log.rs

```diff
@@ -18,37 +22,48 @@ editoast_common::schemas! {
 
 #[derive(Clone, Debug, Serialize, Deserialize, Model, ToSchema)]
 #[model(table = editoast_models::tables::stdcm_logs)]
-#[model(gen(ops = c))]
+#[model(gen(ops = cd, list))]
 pub struct StdcmLog {
     pub id: i64,
     pub trace_id: String,
```

Comment:
```suggestion
	#[model(identifier)]
    pub trace_id: String,
```

Allows you to use Model traits by trace id.

Reply:
We should checked that the `trace_id` column is actually indexed in DB

Reply:
Done.